tools/python_tests/permissions_test.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+
+import logging
+import subprocess
+from argparse import ArgumentParser
+import time
+
+
+EXPECTED_PERMISSIONS = {
+    "uses-permission: android.permission.WRITE_EXTERNAL_STORAGE",
+    "uses-permission: android.permission.ACCESS_COARSE_LOCATION",
+    "uses-permission: android.permission.ACCESS_FINE_LOCATION",
+    "uses-permission: android.permission.INTERNET",
+    "uses-permission: android.permission.ACCESS_WIFI_STATE",
+    "uses-permission: android.permission.CHANGE_WIFI_STATE",
+    "uses-permission: android.permission.ACCESS_NETWORK_STATE",
+    "uses-permission: android.permission.WAKE_LOCK",
+    "uses-permission: android.permission.GET_ACCOUNTS",
+    "uses-permission: com.google.android.c2dm.permission.RECEIVE",
+    "permission: com.mapswithme.maps.pro.beta.permission.C2D_MESSAGE",
+    "uses-permission: com.mapswithme.maps.pro.beta.permission.C2D_MESSAGE",
+    "uses-permission: android.permission.VIBRATE",
+    "permission: com.mapswithme.maps.pro.beta.permission.RECEIVE_ADM_MESSAGE",
+    "uses-permission: com.mapswithme.maps.pro.beta.permission.RECEIVE_ADM_MESSAGE",
+    "uses-permission: com.amazon.device.messaging.permission.RECEIVE",
+}
+
+
+def get_args():
+    parser = ArgumentParser(
+        description="A test for checking that no new permissions have been added to the APK by a third library"
+    )
+
+    parser.add_argument(
+        "-a", "--aapt-path",
+        help="Path to the aapt executable.",
+        dest="aapt", default=None
+    )
+
+    parser.add_argument(
+        "-i", "--input-apk",
+        help="Path to the apk to test.",
+        dest="apk", default=None
+    )
+
+    return parser.parse_args()
+
+
+def exec_shell(executable, flags):
+    spell = ["{0} {1}".format(executable, flags)]
+    logging.info(spell)
+    process = subprocess.Popen(
+        spell,
+        stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+        shell=True
+    )
+
+    out, _ = process.communicate()
+
+    return filter(None, out.splitlines())
+
+
+def find_aapt(args):
+    with open("../../android/local.properties") as props:
```

Comment:
Please, make a relative path from the script, not from the directory the script was executed in.